conda_build/convert.py

```diff
@@ -131,9 +131,8 @@ def extract_temporary_directory(file_path):
     """
     temporary_directory = tempfile.mkdtemp()
 
-    source = tarfile.open(file_path)
-    source.extractall(temporary_directory)
-    source.close()
+    with tarfile.open(file_path) as tar:
+        tar.extractall(temporary_directory, filter="data")
```

Comment:
This extraction is used for converting packages to other archs, is it ok for us to strip group and other's write and execute permissions? See https://docs.python.org/3/library/tarfile.html#tarfile.data_filter

Reply:
The more I think about this I'm thinking this should be changed to the `'tar'` filter

Reply:
this discussion is moot since we delegate to CPH/CPS